recipes/libigl/2.x.x/conanfile.py

```diff
@@ -0,0 +1,103 @@
+import os
+from conans import ConanFile, tools, CMake
+from conans.errors import ConanInvalidConfiguration
+
+
+class LibiglConan(ConanFile):
+    name = "libigl"
+    description = ("Simple C++ geometry processing library")
+    topics = ("conan", "libigl", "geometry", "matrices", "algorithms")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://libigl.github.io/"
+    license = "MPL-2.0"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"header_only": [True, False], "fPIC": [True, False]}
+    default_options = {"header_only": False, "fPIC": True}
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake", "cmake_find_package"
+    requires = ("eigen/3.3.9")
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        tools.check_min_cppstd(self, "14")
```

Comment:
https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#why-is-a-toolscheck_min_cppstd-call-not-enough

Reply:
https://github.com/conan-io/conan-center-index/pull/5542/files/bc35528e45be6b0907c2dcbcc49b1fe42934b1a1#r648484309 is unresolved